tests/acceptance/client/SalesCest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
+namespace hipanel\modules\finance\tests\acceptance\client;
+
+use hipanel\helpers\Url;
+use hipanel\tests\_support\Step\Acceptance\Client;
+
+class AccountSalesCest

```

Comment:
вроде бы по логике других cest'ов этот должен называться просто `SaleCest`
```suggestion
class SaleCest
```

@SilverFire @strorch  ?